README.md

```diff
@@ -12,12 +12,16 @@ Optionally, you may set the overlay to run when the physical switch on the displ
 To install dependencies, you will need [poetry](https://python-poetry.org/docs/#installation) installed on your computer.
 
 At this point in time, you'll also need Python 3.7 installed. Python 3.6 may also work (untested), but right now, the OpenCV version we are using does not have a build available for Python 3.8. [pyenv](https://github.com/pyenv/pyenv) may be used to quickly switch between python versions. To do this,
```

Comment:
Windows-friendly pyenv github link here: https://github.com/pyenv-win/pyenv-win for Windows users (Regular Pyenv doesn't support windows 😔)

Currently bugged, but developer says he is releasing a fix soon